File: Learning-Go/functions1.go
package main

import "fmt"

/*
func FunctionName(param1 type, param2 type, param3 type) {
  // code to be executed
}
*/

func familyName(name string, age int) {
	fmt.Println("Hello", age, "year old", name, "Refsnes")
}

func main() {
	familyName("Liam", 3)
	familyName("Jenny", 14)
	familyName("Anja", 30)
}

File: learn_C++_w3c/functions1.cpp
#include <iostream>
#include <string>
using namespace std;

void myfunction(string fname = "hack", int age = 18) {
    cout << fname << " REfsnes" << age << " years old.\n";
}

int main() {
    myfunction("Liam", 3);
    myfunction("Jonh", 30);
    myfunction();
    return 0;
}